TheMovieDb/Views/BackgroundImageView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BackgroundImageView.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 31/10/21.
+//
+
+import UIKit.UIImageView
+
+final class BackgroundImageView: UIImageView {
+
+    init() {
+        super.init(frame: .zero)
+        image = #imageLiteral(resourceName: "noImage")
+        contentMode = .scaleAspectFill
+        layer.masksToBounds = true
+        backgroundColor = .darkGray
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+}
```

Comment:
this goes against Open/Close principle, you are creating something that does something specific but it's not open for extension, also this seems like a configuration that can be done on a specific view

Reply:
I created that subclass because Im using in 3 diferents cells, I was repeating code, for that reason exist that class

Reply:
just as a Comment look for the **Factory Pattern or Builder Pattern** when building components